.github/workflows/e2e.yml

```diff
@@ -348,19 +358,17 @@ jobs:
           helm repo add jetstack https://charts.jetstack.io
           helm repo update
           helm install certmgr jetstack/cert-manager --set installCRDs=true --version v1.8.0 --namespace cert-manager --create-namespace
-          helm install featureform ./charts/featureform --set global.repo=local --set global.pullPolicy=Never \ 
-          --set global.version=stable --set backup.enabled=true --set backup.interval=1m
+          helm install featureform ./charts/featureform --set global.repo=local --set global.pullPolicy=Never --set global.version=stable --set backup.enabled=true --set backup.schedule="* * * * *"
 
       - name: Create Secret
         env:
-          CLOUD_PROVIDER: "AZURE"
-          AZURE_STORAGE_ACCOUNT: ${{ secrets.AZURE_STORAGE_ACCOUNT }}
-          AZURE_STORAGE_TOKEN: ${{ secrets.AZURE_STORAGE_TOKEN }}
+          AZURE_ACCOUNT_NAME: ${{ secrets.AZURE_ACCOUNT_NAME }}
+          AZURE_STORAGE_KEY: ${{ secrets.AZURE_ACCOUNT_KEY }}
           AZURE_CONTAINER: ${{ secrets.AZURE_CONTAINER_NAME }}
-          STORAGE_PATH: ${{ secrets.STORAGE_PATH }}
+          STORAGE_PATH: "backup"
```

Comment:
This on purpose? I dont understand

Reply:
yeah its so its more visible what the path is for debugging purposes. Otherwise you need to remember or have it documented somewhere since its stored as a secret